runelite-client/src/test/java/net/runelite/client/plugins/opponentinfo/OpponentInfoPluginTest.java

```diff
@@ -0,0 +1,199 @@
+package net.runelite.client.plugins.opponentinfo;
+
+import com.google.inject.Guice;
+import com.google.inject.Inject;
+import com.google.inject.testing.fieldbinder.Bind;
+import com.google.inject.testing.fieldbinder.BoundFieldModule;
+import net.runelite.api.*;
+import net.runelite.api.events.InteractingChanged;
+import net.runelite.client.ui.overlay.OverlayManager;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class OpponentInfoPluginTest
+{
+    @Mock
+    @Bind
+    Client client;
+
+    @Mock
+    @Bind
+    OverlayManager overlayManager;
+
+    @Mock
+    @Bind
+    OpponentInfoConfig opponentInfoConfig;
+
+    @Mock
+    @Bind
+    OpponentInfoOverlay opponentInfoOverlay;
+
+    @Mock
+    @Bind
+    PlayerComparisonOverlay playerComparisonOverlay;
+
+    @Inject
+    OpponentInfoPlugin opponentInfoPlugin;
+
+    @Before
+    public void before()
+    {
+        Guice.createInjector(BoundFieldModule.of(this)).injectMembers(this);
+    }
+
+    /*
+     * Verify that the current opponent in multi is always the npc that the player is attacking
+     */
+    @Test
+    public void testAttackingEnemyInMulti() {
+        Player thePlayer = mock(Player.class), otherPlayer = mock(Player.class);
```

Comment:
We generally use the name `localPlayer` for the local player
```suggestion
        Player localPlayer = mock(Player.class), otherPlayer = mock(Player.class);
```